### ISSUE TITLE
Fine-tuning GBIF models with trap data

### ISSUE THREAD

**Issue by adityajain07** [RolnickLab/ami-ml#20]
Experiments to quantify the improvement in accuracy when AMI-GBIF models fine-tuned with AMI-Traps data

**Comment by adityajain07** [comment]
Experiments:
1. Only AMI-GBIF 
2. Ony AMI-Traps
3. AMI-GBIF fine-tuned with AMI-Traps
4. AMI-GBIF + AMI-Traps combined

To do (in-order):
1. Solve https://github.com/RolnickLab/ami-ml/issues/21
2. Datasplit of AMI-Traps dataset
3. Re-organize and test training code. Push it to src. https://github.com/RolnickLab/ami-ml/issues/23 maybe worked on in this process.
4. Get evaluation metrics for the first three experiments
5. Combining the AMI-GBIF and AMI-Traps dataset, and performing the #4 experiment

**Comment by joseepoirier** [comment]
Following discussion between AJ and DR, this task is downprioritised to work on global model first.
